pkg/pillar/zedUpload/datastore.go

```diff
@@ -29,8 +29,8 @@ const (
 	SyncOpGetURI                = 7
 	SysOpPutPart                = 8
 	SysOpCompleteParts          = 9
-
-	DefaultNumberOfHandlers = 10
+	DefaultNumberOfHandlers     = 10
```

Comment:
Should this be the max of the enum values? If not, please comment why it is not.

Reply:
DefaultNumberOfHandlers = 10 was already existing before, so I just used the next available enum.
will re-order it

Reply:
How is DefaultNumberOfHandlers used? Is it the max?

Reply:
while creating the NewDronaCtx we are using this DefaultNumberOfHandlers (in case user specifies the number of handlers as 0) to specify the buffer size i of the request and the response channel.